File: src/org/r3/db/project/team/TeamDTO.java
//
// GPA Gruppo Progetti Avanzati s.r.l. - 
// Progetto UNEP 2012 per PosteItaliane S.p.a.
//
//
// TODO: Readme Section.
//


package org.r3.db.project.team;

public class TeamDTO extends TeamDTOBase
{
	public enum TeamStatus { team_empty, team_building, team_verifying, team_reporting, team_validating, team_close };
	
    public TeamDTO()
    {
        super();
    }

    public TeamDTO(TeamDTO another)
    {
        super(another);
    }

    public String getBusinesskey()
    {
        StringBuilder stb = new StringBuilder();       
        stb.append(teamid);
        return stb.toString();
    }
}

File: WebContent/org/r3/db/project/team/TeamDTO.js
//
// GPA Gruppo Progetti Avanzati s.r.l. - 
// Progetto UNEP 2012 per PosteItaliane S.p.a.
//
//
// TODO: Readme Section.
//



jQuery.r3Utils.namespace("org.r3.db.project.team");

org.r3.db.project.team.TeamDTO = Class.extend(
{
	 init: function(aDataBean)
	 {  
	 	 	 this.dtrowclass   = "grade_team";
       this.dtokey       = aDataBean.dtokey   || null;
       this.dtodescr     = aDataBean.dtodescr || null;
       
        this.site = aDataBean.site || null;
        this.teamid = aDataBean.teamid || null;
        this.teamdescr = aDataBean.teamdescr || null;
        this.periodindex = aDataBean.periodindex || null;
        this.periodtype = aDataBean.periodtype || null;
        this.teamstatus = aDataBean.teamstatus || null;        
 
	 }
    
  ,compareTo: function(anItemId)
   {
   	   if (!this.dtokey)
   	       return -99;
   	       
   	   if (this.dtokey < anItemId)
   	       return -1;
   	   else if (this.dtokey > anItemId)
   	   	   return  1;
   	   	   
   	   return 0;
   }
   
});
